src/utils/sourceHelper.ts

```diff
@@ -43,8 +40,5 @@ export const sourcesMatch = (
   defaultSource: string | undefined,
   itemSource: string | undefined,
 ): boolean => {
-  return (
-    itemSource === defaultSource ||
-    (defaultSource === manualSourceValue && itemSource === oldManualSourceValue)
-  );
+  return itemSource === defaultSource;
```

Comment:
We no longer want to match it with manualSourceValue?

Reply:
We don't need that anymore because we aren't checking for both the current `manualSourceValue` and the `oldManualSourceValue`.